### DISCUSSION TITLE
Error loading Sentinel-2 images

### DISCUSSION THREAD

**Discussion by thaimunhoz** [microsoft/PlanetaryComputer#334] [Q&A]
Hi,

I'm trying to load some Sentinel-2 images, but I'm getting some errors.

![image](https://github.com/microsoft/PlanetaryComputer/assets/80410170/88756241-1d99-4688-91b2-060e07aaa3d5)
![image](https://github.com/microsoft/PlanetaryComputer/assets/80410170/7bafe780-ad84-444e-b1ad-f63abb405c90)

I tried to set the GDAL_HTTP_MAX_RETRY, but still getting the same error :/

**Comment by TomAugspurger** [comment]
Can you share the STAC item whose assets you're trying to load?

**Reply by thaimunhoz** [reply]
Sure!

 https://sentinel2l2a01.blob.core.windows.net/sentinel2-l2/22/L/DJ/2024/01/15/S2A_MSIL2A_20240115T133831_N0510_R124_T22LDJ_20240115T183726.SAFE/GRANULE/L2A_T22LDJ_A044737_20240115T133826/IMG_DATA/R20m/T22LDJ_20240115T133831_SCL_20m.tif?st=2024-04-15T12%3A19%3A46Z&se=2024-04-16T13%3A04%3A46Z&sp=rl&sv=2021-06-08&sr=c&skoid=c85c15d6-d1ae-42d4-af60-e2ca0f81359b&sktid=72f988bf-86f1-41af-91ab-2d7cd011db47&skt=2024-04-16T11%3A13%3A08Z&ske=2024-04-23T11%3A13%3A08Z&sks=b&skv=2021-06-08&sig=2vFcq4qMAQ/B/5zqrEjfcDFxx0pT27%2BZ9hJu3xrJPP0%3D:1
Aborting load due to failure while reading: https://sentinel2l2a01.blob.core.windows.net/sentinel2-l2/22/L/CJ/2024/01/15/S2A_MSIL2A_20240115T133831_N0510_R124_T22LCJ_20240115T181439.SAFE/GRANULE/L2A_T22LCJ_A044737_20240115T133826/IMG_DATA/R20m/T22LCJ_20240115T133831_SCL_20m.tif?st=2024-04-15T12%3A19%3A46Z&se=2024-04-16T13%3A04%3A46Z&sp=rl&sv=2021-06-08&sr=c&skoid=c85c15d6-d1ae-42d4-af60-e2ca0f81359b&sktid=72f988bf-86f1-41af-91ab-2d7cd011db47&skt=2024-04-16T11%3A13%3A08Z&ske=2024-04-23T11%3A13%3A08Z&sks=b&skv=2021-06-08&sig=2vFcq4qMAQ/B/5zqrEjfcDFxx0pT27%2BZ9hJu3xrJPP0%3D:1
Aborting load due to failure while reading: https://sentinel2l2a01.blob.core.windows.net/sentinel2-l2/22/L/DJ/2024/01/15/S2A_MSIL2A_20240115T133831_N0510_R124_T22LDJ_20240115T183726.SAFE/GRANULE/L2A_T22LDJ_A044737_20240115T133826/IMG_DATA/R20m/T22LDJ_20240115T133831_SCL_20m.tif?st=2024-04-15T12%3A19%3A46Z&se=2024-04-16T13%3A04%3A46Z&sp=rl&sv=2021-06-08&sr=c&skoid=c85c15d6-d1ae-42d4-af60-e2ca0f81359b&sktid=72f988bf-86f1-41af-91ab-2d7cd011db47&skt=2024-04-16T11%3A13%3A08Z&ske=2024-04-23T11%3A13%3A08Z&sks=b&skv=2021-06-08&sig=2vFcq4qMAQ/B/5zqrEjfcDFxx0pT27%2BZ9hJu3xrJPP0%3D:1
Aborting load due to failure while reading: https://sentinel2l2a01.blob.core.windows.net/sentinel2-l2/22/L/CK/2024/01/15/S2A_MSIL2A_20240115T133831_N0510_R124_T22LCK_20240115T181435.SAFE/GRANULE/L2A_T22LCK_A044737_20240115T133826/IMG_DATA/R20m/T22LCK_20240115T133831_SCL_20m.tif?st=2024-04-15T12%3A19%3A46Z&se=2024-04-16T13%3A04%3A46Z&sp=rl&sv=2021-06-08&sr=c&skoid=c85c15d6-d1ae-42d4-af60-e2ca0f81359b&sktid=72f988bf-86f1-41af-91ab-2d7cd011db47&skt=2024-04-16T11%3A13%3A08Z&ske=2024-04-23T11%3A13%3A08Z&sks=b&skv=2021-06-08&sig=2vFcq4qMAQ/B/5zqrEjfcDFxx0pT27%2BZ9hJu3xrJPP0%3D:1
Aborting load due to failure while reading: https://sentinel2l2a01.blob.core.windows.net/sentinel2-l2/22/L/CK/2024/01/15/S2A_MSIL2A_20240115T133831_N0510_R124_T22LCK_20240115T181435.SAFE/GRANULE/L2A_T22LCK_A044737_20240115T133826/IMG_DATA/R20m/T22LCK_20240115T133831_SCL_20m.tif?st=2024-04-15T12%3A19%3A46Z&se=2024-04-16T13%3A04%3A46Z&sp=rl&sv=2021-06-08&sr=c&skoid=c85c15d6-d1ae-42d4-af60-e2ca0f81359b&sktid=72f988bf-86f1-41af-91ab-2d7cd011db47&skt=2024-04-16T11%3A13%3A08Z&ske=2024-04-23T11%3A13%3A08Z&sks=b&skv=2021-06-08&sig=2vFcq4qMAQ/B/5zqrEjfcDFxx0pT27%2BZ9hJu3xrJPP0%3D:1
Aborting load due to failure while reading: https://sentinel2l2a01.blob.core.windows.net/sentinel2-l2/22/L/CJ/2024/01/15/S2A_MSIL2A_20240115T133831_N0510_R124_T22LCJ_20240115T181439.SAFE/GRANULE/L2A_T22LCJ_A044737_20240115T133826/IMG_DATA/R20m/T22LCJ_20240115T133831_SCL_20m.tif?st=2024-04-15T12%3A19%3A46Z&se=2024-04-16T13%3A04%3A46Z&sp=rl&sv=2021-06-08&sr=c&skoid=c85c15d6-d1ae-42d4-af60-e2ca0f81359b&sktid=72f988bf-86f1-41af-91ab-2d7cd011db47&skt=2024-04-16T11%3A13%3A08Z&ske=2024-04-23T11%3A13%3A08Z&sks=b&skv=2021-06-08&sig=2vFcq4qMAQ/B/5zqrEjfcDFxx0pT27%2BZ9hJu3xrJPP0%3D:1
Aborting load due to failure while reading: https://sentinel2l2a01.blob.core.windows.net/sentinel2-l2/22/L/CK/2024/01/15/S2A_MSIL2A_20240115T133831_N0510_R124_T22LCK_20240115T181435.SAFE/GRANULE/L2A_T22LCK_A044737_20240115T133826/IMG_DATA/R20m/T22LCK_20240115T133831_SCL_20m.tif?st=2024-04-15T12%3A19%3A46Z&se=2024-04-16T13%3A04%3A46Z&sp=rl&sv=2021-06-08&sr=c&skoid=c85c15d6-d1ae-42d4-af60-e2ca0f81359b&sktid=72f988bf-86f1-41af-91ab-2d7cd011db47&skt=2024-04-16T11%3A13%3A08Z&ske=2024-04-23T11%3A13%3A08Z&sks=b&skv=2021-06-08&sig=2vFcq4qMAQ/B/5zqrEjfcDFxx0pT27%2BZ9hJu3xrJPP0%3D:1
Aborting load due to failure while reading: https://sentinel2l2a01.blob.core.windows.net/sentinel2-l2/22/L/CK/2024/01/15/S2A_MSIL2A_20240115T133831_N0510_R124_T22LCK_20240115T181435.SAFE/GRANULE/L2A_T22LCK_A044737_20240115T133826/IMG_DATA/R20m/T22LCK_20240115T133831_SCL_20m.tif?st=2024-04-15T12%3A19%3A46Z&se=2024-04-16T13%3A04%3A46Z&sp=rl&sv=2021-06-08&sr=c&skoid=c85c15d6-d1ae-42d4-af60-e2ca0f81359b&sktid=72f988bf-86f1-41af-91ab-2d7cd011db47&skt=2024-04-16T11%3A13%3A08Z&ske=2024-04-23T11%3A13%3A08Z&sks=b&skv=2021-06-08&sig=2vFcq4qMAQ/B/5zqrEjfcDFxx0pT27%2BZ9hJu3xrJPP0%3D:1

**Comment by thaimunhoz** [comment]
Hi, any update on this error? I'm still trying to figure out how to solve it. 

Sometimes the error message changes to this:

ERROR: root: An exception occurred: Read or write failed. IReadBlock failed at X offset 19, Y offset 16: TIFFReadEncodedTile() failed.

Any suggestion is welcome :)

**Reply by lucas-srigley** [reply]
Hi there, did you ever find a solution to your issue? I'm getting the same error
